snuba/datasets/processors/generic_metrics_processor.py

```diff
@@ -146,10 +158,20 @@ def process_message(
                 message["sentry_received_timestamp"]
             )
 
+        self.__record_cogs(message)
         return InsertBatch(
             [processed], None, sentry_received_timestamp=sentry_received_timestamp
         )
 
+    def __record_cogs(self, message: GenericMetric) -> None:
+        if random() < (get_config("gen_metrics_processor_cogs_probability") or 0):
+            record_cogs(
+                resource_id=self._resource_id,
+                app_feature=message["use_case_id"],
```

Comment:
Please prefix the use_case_id with `genericmetrics_`.
Since the use case ids will be owned by product they are likely to eventually conflict with other app_feature labels.